.github/workflows/sterile.yml

```diff
@@ -100,6 +100,12 @@ jobs:
         run: |
           just debug=true rust="${{matrix.rust}}" profile=release target=x86_64-unknown-linux-musl sterile cargo test
 
+      - name: "run clippy"
+        if: ${{ always() }}
+        run: |
+          just debug=true rust="${{matrix.rust}}" profile=dev target=x86_64-unknown-linux-gnu sterile cargo clippy
+
+
```

Comment:
Do you expect Clippy in the sterile build to report errors that it wouldn't raise in dev.yaml?

Reply:
No but this is more of a bisection tool.

If sterile passes and dev rejects or vise versa then we have a basis to debug.  Clippy is normally a fast test so it seems fine here